### DISCUSSION TITLE
Axios is not a function after update

### DISCUSSION THREAD

**Discussion by Nollknolle** [axios/axios#5086] [Q&A]
Hey,
after the update, I get the following error on my requests...
Does anyone know that something got changed.
It worked except for today after the update.

**Error:**
```
   await axios(config)
            ^
TypeError: axios is not a function
```

**My Code:**
```js
const axios = require("axios").default;

var config = {
        method: "get",
        url: "xxx",
        headers: {
          "client-id": "xxx",
          Authorization: `xxx`,
        },
      };

      await axios(config) // <-- ERROR
        .then(function (response) {
         [...]
```

**Comment by Nollknolle** [comment]
Axios Version: **^1.1.2**

**Reply by Nollknolle** [reply]
Downgrade to **1.1.0** is working for me.

**Reply by mpwellen** [reply]
Downgrading to 1.1.2 seems to work for me as well.

**Comment by ralphtheninja** [comment]
@Nollknolle Skip the `.default` in node.

**Reply by renatotr** [reply]
The same occurred with me and the removal of the ".default", parameter made it work just fine, although it's still in the examples of the page.
